lib/src/source/git.dart

```diff
@@ -40,6 +41,49 @@ class GitSource extends CachedSource {
     });
   }
 
+  /// Gets the list of all versions from git tags.
+  /// Anything that Version.parse understands is considered a version,
+  /// it will also attempt to strip off a preceding 'v' e.g. v1.0.0
+  Future<List<Pubspec>> getVersions(String name, description) async {
+    if (!_useVersionTags(description)) {
```

Comment:
The issue I'm having here is that I can't seem to access the `version` in this method, as it is not part of the description.

```
w_transport:
  version: ^1.0.0
  git: git@github.com:Worvia/w_transport.git
```
> description --> "git@github.com:Workiva/w_transport.git"

```
w_transport:
  version: ^1.0.0
  git: 
    url: git@github.com:Worvia/w_transport.git
    use_version_tags: true
```
> description --> {'url': 'git@github.com:Worvia/w_transport.git', 'use_version_tags': true}

Any idea how to short-circuit based on the `version`?